src/kansei/fallen/node/fallen_determinant.cpp

```diff
@@ -31,25 +31,37 @@ namespace kansei::fallen
 {
 
 FallenDeterminant::FallenDeterminant(const DeterminantType & type)
-: fallen_status(FallenStatus::STANDUP), determinant_type(type), fallen_back_raw_limit(491.0),
-  fallen_front_raw_limit(458.0), fallen_right_raw_limit(519.0), fallen_left_raw_limit(498.0)
+: fallen_status(FallenStatus::STANDUP), determinant_type(type)
 {
 }
```

Comment:
Don't let these properties of `FallenDeterminant` class uninitialized:

https://github.com/ichiro-its/kansei/blob/bce6d9ae8130c3e49a9e38a18b49c5d9a767cc68/include/kansei/fallen/node/fallen_determinant.hpp#L54-L63